Assignment4/wnicolait-80097/task06.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1YORntmOcemEVo3vx4xIU0BwvXJ5Tc-3m
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2020-2021/master/Assignment4"
+
+"""Leemos el fichero RDF de la forma que lo hemos venido haciendo"""
+
+from rdflib import Graph, Namespace, Literal
+from rdflib.namespace import RDF, RDFS
+g = Graph()
+g.namespace_manager.bind('nsPer', Namespace("http://Person"), override=False)
+g.namespace_manager.bind('nsP', Namespace("http://Universithy"), override=False)
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/resources/example5.rdf", format="xml")
+
+"""Create a new class named Researcher"""
+
+ns = Namespace("http://somewhere#")
+g.add((ns.Researcher, RDF.type, RDFS.Class))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.1: Create a new class named "University"**"""
+
+nsP = Namespace("http://Universithy/")
+nsPer= Namespace("http://Person/")
+g.add((nsP.Universithy, RDF.type, RDFS.Class))
+g.add((nsPer.Person, RDF.type, RDFS.Class))
+for s, p, o in g:
+  print(s,p,o)
+
+
+
+"""**TASK 6.2: Add "Researcher" as a subclass of "Person"**"""
+
+
+
+app = nsPer.Researcher
+     says= nsPer.says
+     inv=Literal("Jane Smith")
+     print(app)
+
+"""**TASK 6.3: Create a new individual of Researcher named "Jane Smith"**"""
```

Comment:
ufff read carefully the tasks please, these lines are copy-paste from previous tasks